apps/spruce/src/components/Table/utils.ts

```diff
@@ -1,8 +1,6 @@
 import { OnChangeFn } from "@tanstack/react-table";
 
 export enum TableQueryParams {
-  Limit = "limit",
-  Page = "page",
```

Comment:
Any reason you moved these values outside of this enum instead of using this enum for the hook? I feel like it makes sense to group them under one enum, or at least to colocate the enums.

Reply:
I moved these since pagination is not explicitly restricted to tables. We utilize the same component and logic on the patches pages so it didn't make sense to tie the pagination query params to the table component. I was hoping I could utilize some sort of inheritance with these 2 fields but it is not supported with enums. I could colocate them in the `queryParms` file if that makes sense?

Reply:
Ah good point! And I've also tried that before to the same result 😢 I think the `queryParams` file sounds perfect!